tests/Unit/NumericalAlgorithms/LinearOperators/Python/Test_PowerMonitors.py

```diff
@@ -36,6 +37,38 @@ def test_power_monitors(self):
 
         np.testing.assert_allclose(np_test_array, np_check_array, 1e-12, 1e-12)
 
+    # Check that the truncation error for a constant unit function is
+    # consistent with numerical floor when we use a high order basis
+    def test_relative_truncation_error(self):
+        num_points_per_dimension = 3
+
+        extent = num_points_per_dimension
+        basis = Basis.Legendre
+        quadrature = Quadrature.GaussLobatto
+        mesh = Mesh2D(extent, basis, quadrature)
+
+        np_vec = np.ones(mesh.number_of_grid_points())
+        test_vec = DataVector(np_vec)
```

Comment:
You don't need to do this, the Numpy array converts to a DataVector automatically